sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/HttpHelper.cs

```diff
@@ -68,5 +68,33 @@ internal static string GetUrl(Dictionary<string, string> tags)
 
             return string.IsNullOrWhiteSpace(url) ? null : url;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static string GetHttpStatusCode(Dictionary<string, string> tags)
+        {
+            if (tags != null && tags.TryGetValue(SemanticConventions.AttributeHttpStatusCode, out var status))
+            {
```

Comment:
Interestingly enough this is just different than the previous comment where the dictionary is checked for null and the try get executed, so now helpers have difference semantics.

Reply:
Good catch :) Changed previous method to match the semantics.